### ISSUE TITLE
[Feature Request] Delete standard issue modifications

### ISSUE THREAD

**Issue by Terranaut** [chummer5a/chummer5a#2881]
(Please fill out the below as accurately as possible. More detail makes it easier for us to find the problems and fix them.)

### Environment
Chummer Version: Nightly
Environment: -
Runtime: -

Currently it's not possible to remove items/modifications that come, for example, vehicles.
Example: he F-B Bumblebee is very attractive to Riggers for multiple reasons, but the Machine Gun it comes with is not always the optimal choice. It is, in theory, possible to refit it with other Heavy Weapons, for example Grenade or Missile Launchers. But as it is not possible to delete the MG from the Weapon Mount, this is awkward to represent.

tl;dr: Please add the option to remove default modifcations from items such as drones and vehicles.

**Comment by chummer5a** [comment]
No real reason we can't do it, it's largely just a question of whether to provide some kind of support for re-adding standard mods and such.

**Comment by Haplo164** [comment]
would it be possible to add an "equipped" property to the mods that frees up the slot?
